welcome-message/index.js

```diff
@@ -1,22 +1,103 @@
 const superstore = require('superstore-sync');
-
-import { $ } from '../utils';
+const expander = require('o-expander');
+const oViewport = require('o-viewport');
 
 const STORAGE_KEY = 'n-welcome-message-seen';
+const HAS_MINIMIZED = 'n-welcome-message-collapsed';
 const TEST_KEY = 'n-welcome-message-test';
 const TEST_VAL = 'can-store';
 
+let fixedEl;
+let fixedElHeight;
+let staticEl;
+let viewportHeight;
+
 function hasLocalStorage () {
 	superstore.local.set(TEST_KEY, TEST_VAL);
 	const retrieved = superstore.local.get(TEST_KEY);
 	superstore.local.unset(TEST_KEY);
 	return TEST_VAL === retrieved && superstore.isPersisting();
 }
```

Comment:
I think it defaults to true and gets updated when you get or set

https://github.com/matthew-andrews/superstore-sync/blob/master/lib/superstore-sync.js#L14

https://github.com/matthew-andrews/superstore-sync/blob/master/lib/superstore-sync.js#L128